.infrastructure/mysql/mysql-secret.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysql-secret
+  namespace: mysql
+type: Opaque
+data:
+    MYSQL_ROOT_PASSWORD: "cm9vdA==" #Base64 "root"
```

Comment:
Using 'root' as the root password is not secure. Consider using a stronger, more complex password.